packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts

```diff
@@ -101,7 +101,7 @@ describe('JSONSchema generator', () => {
             meaning: {
               type: 'number',
               'x-faker': {
-                'random.number': {
+                'datatype.number': {
```

Comment:
we'd like to ensure that the 5.x behavior is utilized - can this be reverted?

Reply:
@daniel-white done